cg/server/admin.py

```diff
@@ -143,14 +160,20 @@ class ApplicationView(BaseView):
         "category",
     ]
     column_formatters = {
+        "order_types": view_order_types,
         "sample_concentration_minimum": view_sample_concentration_minimum,
         "sample_concentration_maximum": view_sample_concentration_maximum,
         "sample_concentration_minimum_cfdna": view_sample_concentration_minimum_cfdna,
         "sample_concentration_maximum_cfdna": view_sample_concentration_maximum_cfdna,
     }
     column_filters = ["prep_category", "is_accredited"]
     column_searchable_list = ["tag", "prep_category"]
-    form_excluded_columns = ["category", "versions"]
+    form_excluded_columns = ["category", "versions", "order_types"]
```

Comment:
Apparently relationships are by default shown in the edit/create form in flask, meaning that we need to exclude the default order types

Reply:
And instead add them via tick boxes in the form_extra_fields